app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
+    people = {}
```

Comment:
The `people` dictionary is used to store `Person` instances by their name. Ensure that names are unique to avoid overwriting existing entries.